pantheon/src/main/java/tech/pegasys/pantheon/cli/PantheonCommand.java

```diff
@@ -709,4 +712,12 @@ private Path dataDir() {
   private boolean isFullInstantiation() {
     return !isDocker;
   }
+
+  private MetricsSystem supplyMetricSystem() {
+    if (rpcApis.contains(RpcApis.METRICS)) {
```

Comment:
This is an optimization that will need more thought, I created NC-2074 to cover this.

As for this patch, both the JSON-RPC debug_metrics api and the prometheus endpoint are exposed at the same time.  We get both or we get none.

Reply:
Why not just give the user control of enabling metrics with a flag?

Can you remove the METRICS rpc option since that doesn't follow the convention?